static-assets/components/cstudio-forms/controls/rte-tinymce5.js

```diff
@@ -15,625 +15,663 @@
  */
 
 CStudioForms.Controls.RTETINYMCE5 = CStudioForms.Controls.RTETINYMCE5 ||
-function(id, form, owner, properties, constraints, readonly, pencilMode)  {
-	this.owner = owner;
-	this.owner.registerField(this);
-	this.errors = [];
-	this.properties = properties;
-	this.constraints = constraints;
-	this.inputEl = null;
-	this.required = false;
-	this.value = "_not-set";
-	this.form = form;
-	this.id = id;
-	this.readonly = readonly;
-	this.rteHeight = 300;
-  this.pencilMode = pencilMode;
-  this.supportedPostFixes = ["_html"];
-
-	return this;
-}
+  function (id, form, owner, properties, constraints, readonly, pencilMode) {
+    this.owner = owner;
+    this.owner.registerField(this);
+    this.errors = [];
+    this.properties = properties;
+    this.constraints = constraints;
+    this.inputEl = null;
+    this.required = false;
+    this.value = '_not-set';
+    this.form = form;
+    this.id = id;
+    this.readonly = readonly;
+    this.rteHeight = 300;
+    this.pencilMode = pencilMode;
+    this.supportedPostFixes = ['_html'];
+
+    this.messages = {
+      rteControlMessages: CrafterCMSNext.i18n.messages.rteControlMessages
+    };
```

Comment:
`this.messages = CrafterCMSNext.i18n.messages.rteControlMessages`

Notice this changes messages. Instead of using messages.rteControlMessages, just access the internal props of rteControlMessages.